app/elements/elements.html

```diff
@@ -59,6 +59,10 @@
 <!-- /* END-REMOVE-IMPORTS */ -->
 
 <!-- Add your elements here -->
+<!-- Socobo Behaviors Elements -->
+<link rel="import" href="socobo-element-toast/socobo-element-toast-send-behavior.html">
+<link rel="import" href="socobo-recipe/socobo-recipe-validate-behavior.html">
+
```

Comment:
Maybe the other behavior imports could be moved here as well. Also, the demos do not work because the imports are missing